src/tasks/replaceGlobals.ts

```diff
@@ -660,6 +660,11 @@ async function migrate(args: Mod.MigrateArguments): Promise<boolean> {
 
 		// Try to replace the call
 		try {
+			// if there is no replacer configured error out
+			if (!mReplacerFuncs[oReplace.import.replacerName]) {
+				throw Error(`replacement ignored, no replacer configured for ${
```

Comment:
Maybe instead of "no replacer configured" I would say "no replacer available at the moment for ...".

Not configured sounds like the user has missed some configuration, but this is rather internally.

Reply:
it is missing a replacer -> so this error message is appropriate and should be fixed by the user